cli/src/config.ts

```diff
@@ -500,7 +500,9 @@ async function determineGemfileOrCocoapodPath(
     if (!gemfileText) {
       return 'pod';
     }
-    const cocoapodsInGemfile = new RegExp(/gem 'cocoapods'/).test(gemfileText);
+    const cocoapodsInGemfile = new RegExp(/gem\s+['"]cocoapods/).test(
```

Comment:
This doesn't test for the existence of an end quote. Perhaps something more like: `/gem\s('cocoapods'|"cocoapods")/` that way it test not only existence of both quotes but that the quotes match on each end

Reply:
Yeah I left that out on purpose just to match, but I guess if it's missing, there is an issue. I don't care so much that it's syntactically correct, just that I see it in there. If the Gemfile syntax is borked, someone needs to fix it.

Reply:
Also, I added `\s+` so it works no matter how much space you put on the line. I ran it with a bunch of space between gem and cocoapods and it worked.

Reply:
So I added a closing character class, they may not match but I don't know how much I care about that. `bundle exec` won't run this file.

Reply:
I still believe checking for the quotes and that they match would be good, the `\s+` makes sense if gem doesn't care about extra whitespace. However as long as this works its all good either way